client/src/pages/Landing/useStyles.ts

```diff
@@ -0,0 +1,32 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+    minWidth: '100vw',
+    display: 'flex',
+  },
+  column: {
+    width: '50vw',
+  },
+  dogs: {
+    width: '100%',
+    height: '100%',
+  },
+  authWrapper: {
+    display: 'flex',
```

Comment:
You can apply any significant styles to the Box component directly

Reply:
Thanks for your reminder. Done.